main/src/addins/VersionControl/MonoDevelop.VersionControl/MonoDevelop.VersionControl/Repository.cs

```diff
@@ -258,7 +262,7 @@ public IEnumerable<VersionInfo> GetVersionInfo (IEnumerable<FilePath> paths, Ver
 				else {
 					// If there is no cached status, query it asynchronously
 					vi = new VersionInfo (p, "", Directory.Exists (p), VersionStatus.Versioned, null, VersionStatus.Versioned, null);
-					infoCache.SetStatus (vi, false);
+					infoCache.SetStatus (vi, true);
```

Comment:
```suggestion
					infoCache.SetStatus (vi, false);
```
This change shouldn't be required, the queue processor will pick this up and notify (that was broken, but now fixed with the other changes in this PR)